torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -31,90 +31,113 @@ bool isSupported(Node* node) {
   if (tensorexpr::getTEGenerateBlockCode()) {
     return isSupportedForBlock(node);
   }
-  // TODO:
+
+  static const OperatorSet supported_operator_set{
+      "aten::add.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor",
```

Comment:
The operator order & comments are preserved from the original set of node kinds so it should be easy to review this